pet-kata-solutions/src/test/java/org/eclipse/collections/petkata/Exercise4Test.java

```diff
@@ -30,31 +31,24 @@
 public class Exercise4Test extends PetDomainForKata {
     @Test
     public void getAgeStatisticsOfPets() {
-        // Try to use a MutableIntList here instead
-        // Hints: flatMap = flatCollect, map = collect, mapToInt = collectInt
-        MutableIntList petAges = this.people
+        var petAges = this.people
                 .flatCollect(Person::getPets)
                 .collectInt(Pet::getAge);
 
-        // Try to use an IntSet here instead
         IntSet uniqueAges = petAges.toSet();
 
-        // IntSummaryStatistics is a class in JDK 8 - Try and use it with MutableIntList.forEach()
-        IntSummaryStatistics stats = new IntSummaryStatistics();
+        var stats = new IntSummaryStatistics();
```

Comment:
This can be simplified to:

`var stats = petAges.summaryStatistics();`

Reply:
Fixed